### PR TITLE
[TIMOB-20213] iOS: app crash when closing the active window while sea…

### PR DESCRIPTION
**Pull request by sriks**
JIRA https://jira.appcelerator.org/browse/TIMOB-20213

### CODE REVIEW

File: iphone/Classes/TiUIListView.m [hunk 1]
```diff
@@ -92,6 +92,7 @@ - (void)dealloc
     [_headerViewProxy setProxyObserver:nil];
     [_footerViewProxy setProxyObserver:nil];
     [_pullViewProxy setProxyObserver:nil];
+    searchController.delegate = nil;
```
Comment (pec1985) [inline iphone/Classes/TiUIListView.m]: Use space notation for setters rather than dots.
`[searchController setDelegate:nil];`

**Comment by AngelkPetkov** [comment]
CR and FT approved. Ones Pedro's comment is addressed ill merge the PR. Thanks Srikanth.

File: iphone/Classes/TiUIListView.m (hunk 1 above)
Comment (sriks) [inline iphone/Classes/TiUIListView.m]: Addressed.

**Comment by AngelkPetkov** [comment]
Thanks! Merging

**Pull request merged**